release-notes/Release-Notes-RC2.md

```diff
@@ -0,0 +1,123 @@
+# Release Notes
+
+## RC2 released 5/16/2016
+
+Full details on this release can be read about in this blog post:
+** [the blog title](http://someurl) **
+
+Platform support has been expanded for RC2
+
+* Red Hat Enterprise Linux 7.2
+* Debian 8.2 (8.2, 8.3, 8.4)+
+* Ubuntu 14.04 (16.04 support is coming in RTM)
+* OS X 10.11
+* Centos 7.1
+* Windows 7+ / Server 2012 R2+
+* Windows Nano Server TP5
+
+### Dependencies
+
+- [ ] Are there changes from RC1?
+- [ ] Should min versions of tools be included?
+
+.NET Core RC1 on Linux and OSX take dependencies on the following libraries:
+* libc, librt: I/O, process management, time, etc.
+* libunwind: call chain determination for exception handling and stack traces
+* gettext: resource strings
+* libicu: globalization
+* liblttng: runtime tracing
+* libuuid: guid generation
+* libcurl: HTTP
+* libssl & libcrypto: cryptography
+* libz: deflate and gzip
+* procfs & Core Foundation / Services: process and networking information on Linux and OSX
+
+### Introduction of .NET Core API Documentation
+
+* API: http://dotnet.github.io/api/
+* Conceptual: http://dotnet.github.io/docs
+
+### Runtime
+
+Most of the Runtime work since RC1 has been focused on performance and reliability improvements. A few highlights include:
+
+* [Runtime Configuration](https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/specs/runtime-configuration-file.md) enhancements
+* Server GC on Linux platforms
+
+### JIT
+
+* Port of all JIT changes made for .NET Framework 4.6.1
+* SIMD code generation for System.Numerics on all platforms
+* Code quality improvements
+
+### BCL
+
+Below is a summary of API changes made since RC1. A complete and detailed listing of API changes can be seen [here](https://github.com/dotnet/core/tree/master/release-notes/RC1-RC2_API_diff.md).
+
+New .NET Core APIs
+
+* System.Drawing
+* System.IO.BufferedStream
+* System.IO.Packaging
+
+APIs added in RC2 to existing classes and namespaces
+
+* System.Console
+* System.Diagnostics.ProcessStartInfo
+* System.Environment
+* System.Linq
+* System.Reflection
+* System.Runtime.InteropServices
+* System.Runtime.InteropServices.RuntimeInformation
+* System.Security.Cryptography
+    * Aes, HMACMD5, HMACSHA1, HMACSHA256, HMACSHA384, HMACSHA512, RSA, RSACryptoServiceProvider
```

Comment:
I'm about to send you the crypto RC1->RC2 release notes.  Yeah, I know, they're late...